samples/nextjs/sitecore/config/JssNextWeb.config

```diff
@@ -35,7 +35,9 @@
             hostName="JssNextWeb"
             rootPath="/sitecore/content/JssNextWeb"
             startItem="/home"
-            database="master" />
+            database="master"
+            serverSideRenderingEngine="http"
+            serverSideRenderingEngineEndpointUrl="http://localhost:3000" />
```

Comment:
Is there a way to centralize this URL at all? Maybe not a big deal given the URL param to change it in Experience Editor.

Reply:
@nickwesselman In a docker world, this could be an environment variable, but I'm not sure how to centralize it more in this context? And exactly, you can always override it using the URL param (which you may do quite often if using a dynamic ngrok tunnel in dev e.g. `&sc_httprenderengineurl=https://8eda01470256.ngrok.io`)

Reply:
@ambrauer Environment variable is a good idea. Should we at least add some comments that alert the dev they need to change `serverSideRenderingEngineEndpointUrl` if they change the Editing Host URL or port?

Reply:
@nickwesselman Yeah we should. Added specific callout for `serverSideRenderingEngineEndpointUrl` in [editing server comments](https://github.com/Sitecore/jss/blob/6c3a6df45c5949bb3cecf43da815bc0c8772d3cc/samples/nextjs/scripts/editing-server.ts#L9).